dev/bin/common.sh

```diff
@@ -0,0 +1,229 @@
+credentials_dir="${dev_dir}/credentials"
+kustomize_deploy_dir="${local_deploy_dir}/kustomize"
+local_env_file="${local_deploy_dir}/local_env"
+local_config_file="${dev_config_dir}/localconfig"
+kustomize_dir="${root_dir}/kustomize"
+
+fleet_manager_container_name=${FLEET_MANAGER_CONTAINER_NAME:-'openbridge/manager:latest'}
+fleet_shard_container_name=${FLEET_SHARD_CONTAINER_NAME:-'openbridge/shard-operator:latest'}
+executor_container_name=${EXECUTOR_CONTAINER_NAME:-'openbridge/executor:latest'}
+ingress_container_name=${INGRESS_CONTAINER_NAME:-'openbridge/ingress:latest'}
+
+function configure_kafka {
+  retrieve_full_env
+  check_required_variable "MANAGED_KAFKA_INSTANCE_NAME"
+  manager_kafka_credentials_file="${credentials_dir}/${MANAGED_KAFKA_INSTANCE_NAME}.json"
+  admin_sa_name="${MANAGED_KAFKA_INSTANCE_NAME}-admin"
+  admin_sa_credentials_file="${credentials_dir}/${admin_sa_name}.json"
+  ops_sa_name="${MANAGED_KAFKA_INSTANCE_NAME}-ops"
+  ops_sa_credentials_file="${credentials_dir}/${ops_sa_name}.json"
+  mc_sa_name="${MANAGED_KAFKA_INSTANCE_NAME}-mc"
+  mc_sa_credentials_file="${credentials_dir}/${mc_sa_name}.json"
+}
+
+function configure_managed_connectors {
+  retrieve_full_env
+  check_required_variable "OPENSHIFT_OFFLINE_TOKEN"
+  managed_connectors_cluster_id="${MANAGED_CONNECTORS_CLUSTER_ID:-empty}"
+}
+
+function configure_cluster {
+  retrieve_full_env
+  cluster_type=${CLUSTER_TYPE:-minikube}
+  if [ "${cluster_type}" = 'minikube' ]; then
+    configure_minikube
+  elif [ "${cluster_type}" = 'kind' ]; then
+    configure_kind
+  fi
+}
+
+function configure_minikube {
+  retrieve_full_env
+  minikube_profile="${MINIKUBE_PROFILE:-minikube}"
+  minikube_cpus="${MINIKUBE_CPUS:-4}"
+  minikube_memory="${MINIKUBE_MEMORY:-8192}"
+  minikube_kubernetes_version="${MINIKUBE_KUBERNETES_VERSION:-v1.21.6}"
+  minikube_driver="${MINIKUBE_DRIVER}"
+  minikube_container_runtime="${MINIKUBE_CONTAINER_RUNTIME}"
+}
+
+function configure_kind {
+  retrieve_full_env
+  kind_name="${KIND_NAME:-kind}"
+  kind_config_file="${KIND_CONFIG_FILE:-"${dev_config_dir}/kind/config.yaml"}"
+  kind_container_engine="${KIND_CONTAINER_ENGINE}"
+  kind_network="${KIND_NETWORK}"
+  kind_kubernetes_version="${KIND_KUBERNETES_VERSION:-v1.21.10}"
+  
+}
+
+function configure_cluster_started {
+  retrieve_full_env
+  configure_cluster
+  cluster_ip=${CLUSTER_IP}
+  ping -c 1 ${cluster_ip} &> /dev/null || die "cluster is not responding to ping. Is it started?"
+}
+
+function configure_keycloak {
+  retrieve_full_env
+  keycloak_url=${KEYCLOAK_URL}
+  is_keycloak_internal_url=${IS_KEYCLOAK_INTERNAL_URL}
+  keycloak_username=${KEYCLOAK_USERNAME:-webhook-robot-1}
+  keycloak_password=${KEYCLOAK_PASSWORD:-therobot}
+  keycloak_client_id=${KEYCLOAK_CLIENT_ID:-event-bridge}
+  keycloak_client_secret=${KEYCLOAK_CLIENT_SECRET:-secret}
+  keycloak_token_offline_access=${KEYCLOAK_TOKEN_OFFLINE_ACCESS:-true}
+}
+
+function configure_manager {
```

Comment:
In many scripts I see the usage of `configure_manager_url`. Is it the same? (and btw, where is defined?)

Reply:
gonna check